arrow-libs/core/arrow-core/build.gradle.kts

```diff
@@ -14,7 +14,7 @@ kotlin {
       dependencies {
         api(projects.arrowContinuations)
         api(projects.arrowAnnotations)
-        implementation(libs.kotlin.stdlibCommon)
+        api(libs.kotlin.stdlibCommon)
```

Comment:
@JavierSegoviaCordoba do you know if this is correct.
Since we expose types from Kotlin Std from public API, it should be marked as `api` right?